integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java

```diff
@@ -359,6 +365,14 @@ public static boolean isFuseInstalled() {
     return false;
   }
 
+  /**
+   * @return true if os version is matched to use macFuse
+   */
+  public static boolean isMacFuse() {
+    return new ComparableVersion(SystemUtils.OS_VERSION)
```

Comment:
if `comparable version` requires a new dependency, then may be worth using the original approach.

Reply:
sorry for the extra work introduced. I thought probably in the code base there always some version comparison methods included

Reply:
I reverted to the original method.